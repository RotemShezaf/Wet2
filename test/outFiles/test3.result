addCostumer: SUCCESS
addCostumer: SUCCESS
addCostumer: SUCCESS
addCostumer: SUCCESS
addCostumer: INVALID_INPUT
addCostumer: SUCCESS
addCostumer: SUCCESS
addCostumer: INVALID_INPUT
addCostumer: ALREADY_EXISTS
makeMember: SUCCESS
makeMember: SUCCESS
makeMember: SUCCESS
makeMember: SUCCESS
makeMember: SUCCESS
makeMember: DOESNT_EXISTS
addPrize: SUCCESS
addPrize: SUCCESS
addPrize: SUCCESS
addPrize: INVALID_INPUT
addPrize: INVALID_INPUT
addPrize: INVALID_INPUT
getExpenses: -8
getExpenses: -8
getExpenses: -5
getExpenses: 0
getExpenses: FAILURE
addPrize: SUCCESS
getExpenses: -15
getExpenses: -15
getExpenses: -15
getExpenses: -12
getExpenses: 0
getExpenses: FAILURE
getExpenses: INVALID_INPUT
getExpenses: FAILURE
