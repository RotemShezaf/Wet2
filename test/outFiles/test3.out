addCostumer: SUCCESS
addCostumer: SUCCESS
addCostumer: SUCCESS
addCostumer: SUCCESS
addCostumer: INVALID_INPUT
addCostumer: SUCCESS
addCostumer: SUCCESS
addCostumer: INVALID_INPUT
addCostumer: ALREADY_EXISTS
makeMember: SUCCESS
makeMember: SUCCESS
makeMember: SUCCESS
makeMember: SUCCESS
makeMember: SUCCESS
makeMember: DOESNT_EXISTS
addPrize: SUCCESS
addPrize: SUCCESS
addPrize: SUCCESS
addPrize: INVALID_INPUT
addPrize: INVALID_INPUT
addPrize: INVALID_INPUT
getExpenses: -8
getExpenses: -8
getExpenses: -5
getExpenses: 0
getExpenses: DOESNT_EXISTS
addPrize: SUCCESS
getExpenses: -15
getExpenses: -15
getExpenses: -15
getExpenses: -12
getExpenses: 0
getExpenses: DOESNT_EXISTS
getExpenses: INVALID_INPUT
getExpenses: DOESNT_EXISTS
